### PR TITLE
Ceph: prepare pvc just before osd on pvc test

### PR DESCRIPTION
**Pull request by satoru-takeuchi**
**Description of your changes:**

The PVCs for OSD on PVC test is perepared before running the integration test. In addition, users are forced to run this script for this by themselves. To avoid this problem, running this script inside the integration test.

**Which issue is resolved by this Pull Request:**

Nothing

**Checklist:**

- [o] **CommitLint Bot**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [o] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [o] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [o] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [o] Documentation has been updated, if necessary.
- [o] Unit tests have been added, if necessary.
- [o] Integration tests have been added, if necessary.
- [o] Pending release notes updated with breaking and/or notable changes, if necessary.
- [o] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [o] Code generation (`make codegen`) has been run to update object specifications, if necessary.

> This PR has already passed CI and the last reflection of review results is only about the commit message. So CI can be skipped here.

[skip ci]

### CODE REVIEW

File: tests/framework/installer/environment.go [hunk 1]
```diff
@@ -55,6 +55,11 @@ func baseTestDir() string {
 	return val
 }
 
+// testScratchDevice get the scratch device to be used for OSD
```
Comment (leseb) [inline tests/framework/installer/environment.go:58]:
```suggestion
// TestScratchDevice get the scratch device to be used for OSD
```

Comment (satoru-takeuchi) [reply tests/framework/installer/environment.go]: Oops, sorry. Fixed.

**Comment by leseb** [comment]
Still a typo in the commit message...

**Comment by satoru-takeuchi** [comment]
> Still a typo in the commit message...

Sorry, I overlooked this comment. Fixed.

**Pull request merged**